deploy/packages/python-packages.yaml

```diff
@@ -76,7 +77,6 @@ packages:
       # - py-neo
       # - py-networkx
       # - py-nose
-      - py-munkres
```

Comment:
Ah sorry! Shouldn't have removed this line. `munkres` can stay in here, and the blacklist should make sure that it's not build for py2.

Reply:
Serves me right for blindly copying the diff, without listening to own doubts. :)